### PR TITLE
*: bump to 2.0.1-beta

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ See also [TiDB change log][tidb_change_log] and [PD change log][pd_change_log].
 [tidb_change_log]: https://github.com/pingcap/tidb/blob/master/CHANGELOG.md
 [pd_change_log]: https://github.com/pingcap/pd/blob/master/CHANGELOG.md
 
+## [2.0.1] - 2018-05-10
```
Comment (disksing) [inline CHANGELOG.md]: 2018-05-11?

Comment (BusyJay) [reply CHANGELOG.md]: Actually the day is not settle down yet, I'm going to remove it.

**Comment by disksing** [comment]
LGTM.

**Comment by coocood** [comment]
LGTM

**Comment by BusyJay** [comment]
/run-all-tests tidb=release-2.0 pd=release-2.0 tidb-test=release-2.0

**Pull request merged**